test/org/sagebionetworks/bridge/services/ParticipantServiceTest.java

```diff
@@ -375,6 +387,23 @@ public void createParticipantEmailEnabledVerificationWanted() {
         verify(account).setStatus(AccountStatus.UNVERIFIED);
     }
     
+    @Test
+    public void createParticipantWithPhoneEmailEnabledVerificationWanted() {
+        STUDY.setEmailVerificationEnabled(true);
+        mockHealthCodeAndAccountRetrieval(null, null);
+
+        StudyParticipant phoneOnly = new StudyParticipant.Builder()
+                .copyOf(PARTICIPANT).withEmail(null).build();
+        
+        participantService.createParticipant(STUDY, CALLER_ROLES, phoneOnly, true);
```

Comment:
What about the inverse case, where we don't provide an email and we _don't_ request email verification?

Reply:
Added.